packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Markers as Gmarkers } from '@rentpath/react-ui-core'
+import { markerIcon, markerIconHover } from './markerIcons'
+
+const NOOP = () => { }
+
+export default class Markers extends PureComponent {
+  static propTypes = {
+    marker: PropTypes.func,
+  }
+
+  static defaultProps = {
+    marker: NOOP,
+  }
+
+  get marker() {
+    const originalIcon = markerIcon()
+
+    return features => ({ // eslint-disable-line no-unused-vars
+      icon: originalIcon,
+      onMouseOver: (event, props, marker) => {
+        marker.setIcon(markerIconHover())
+      },
+      onMouseOut: (event, props, marker) => {
+        marker.setIcon(originalIcon)
+      },
+      ...this.props.marker(),
+    })
+  }
+
+  render() {
+    return (
+      <Gmarkers
+        marker={this.marker}
+        {...this.props}
```

Comment:
does the order of these 2 lines need switched? You have a `marker` prop, which I think will override the `marker={this.marker}`, right?

Reply:
Good catch. It should be destructured from this.props instead 👍